tools/pipeline-witness/Azure.Sdk.Tools.PipelineWitness/BlobUploadProcessor.cs

```diff
@@ -887,6 +889,63 @@ private async Task UploadTestRunBlobAsync(string account, Build build, TestRun t
             }
         }
 
+        private async Task UploadTestRunResultsAsync(string account, Build build, TestRun testRun)
+        {
+            try
+            {
+                var continuationToken = string.Empty;
+                var buildIds = new[] { build.Id };
+                var discoveredTestResults = new List<ShallowTestCaseResult>();
+                var blobPath = $"{build.Project.Name}/{testRun.CompletedDate:yyyy/MM/dd}/{testRun.Id}/results.jsonl";
```

Comment:
We have a 1:1 map from containers to tables.  We should make a new container for the blobs and use the same path as the run:

```c#
                var blobPath = $"{build.Project.Name}/{testRun.CompletedDate:yyyy/MM/dd}/{testRun.Id}.jsonl";
                var blobClient = this.testResultsContainerClient.GetBlobClient(blobPath);
```

Reply:
aye aye